charts/applicationset-progressive-sync/README.md

```diff
@@ -26,8 +26,10 @@ Users of Helm v2 should use `--set installCRDs=true` to install the CRDs.
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
+| args.argoNamespace | string | `"argocd"` |  |
 | args.enableLeaderElection | bool | `false` |  |
 | args.metricsAddr | string | `":8080"` |  |
+| args.namespace | string | `"argocd"` |  |
```

Comment:
we should leverage the builtin Release.Namespace?
https://helm.sh/docs/chart_template_guide/builtin_objects/

Also We should add some documentation

Reply:
I don't think we can use an helm object as value. Again, I think I'm missing your suggestion